### PR TITLE
Add user id check for clan invites

### PR DESCRIPTION
**Pull request by L-e-x-o-n**

### CODE REVIEW

File: lib/teiserver_web/controllers/clans/clan_controller.ex [hunk 1]
```diff
@@ -259,6 +259,11 @@ defmodule TeiserverWeb.Clans.ClanController do
     clan = Clans.get_clan!(clan_id)
 
     cond do
+      user_id == nil ->
+        conn
+        |> put_flash(:danger, "User id missing")
```
Comment (Teifion) [inline lib/teiserver_web/controllers/clans/clan_controller.ex:264]: It's going to be seen by non-developers, I'd rather something more like "Unable to find that user"

**Comment by Teifion** [comment]
I like it

**Comment by Teifion** [comment]
Addresses #67

File: lib/teiserver_web/controllers/clans/clan_controller.ex (hunk 1 above)
Comment (L-e-x-o-n) [reply lib/teiserver_web/controllers/clans/clan_controller.ex]: Changed to "Unable to find user"

**Pull request merged**